### DISCUSSION TITLE
how the text is divided into paragraphs

### DISCUSSION THREAD

**Discussion by wesamalnabki** [mindee/doctr#1587] [Q&A]
I'm trying to OCR the text in the image below. The OCR worked perfectly except that the order of the underlines words was wrong. So, the model detected the first line but missed "sus asesores." and in the second paragraph it detected the whole text except "internacional". 
These two words were detected by as a standalone text blocks.

My question is why the model detected them as full sentences. how to change this behavior ? 

![image](https://github.com/mindee/doctr/assets/6717881/816da421-0847-4df9-9c7b-a01bc5b753aa)

**Comment by felixdittrich92** [comment]
Hi @wesamalnabki 👋 
You could pass `resolve_blocks=False` to the `ocr_predictor` instance :)
